components/Modal/Questionnaire/index.vue

```diff
@@ -0,0 +1,135 @@
+l<template>
```

Comment:
this looks like a typo?

Reply:
😆 yes i think it's a typo

Reply:
Changes in [here](https://github.com/jabardigitalservice/dedi-landing-page/pull/80/commits/b25313353bd2a531784c76044e4ad782e293857b)